package/obs/rmt-server.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Tue May 19 12:31:53 UTC 2020 - Luís Caparroz <lcaparroz@suse.com>
+
+- Version 2.5.8
+- Use repomd_parser gem to remove repository metadata parsing code
+  (GitHub issue gh#233).
```

Comment:
This is not a valid way to reference an issue in OBS. Either leave `gh#233` out completely and open a maintenance request when there another change file entry that has a valid identifier (`bsc#...`), or file a bug in Bugzilla and reference its number.

Reply:
Thinking about it, this change is not visible to customers, so not worth submitting a maintennance release for it. So we could rely on the next change to refer a proper bug?

Reply:
@digitaltom yep, that should be fine.

Reply:
I removed the GitHub issue reference from the changelog (0474241).